overlord/servicestate/servicestate.go

```diff
@@ -49,9 +49,28 @@ type Instruction struct {
 
 type ServiceActionConflictError struct{ error }
 
+func computeExplicitServices(appInfos []*snap.AppInfo, names []string) map[string][]string {
+	explicitServices := make(map[string][]string, len(appInfos))
+	requested := make(map[string]bool)
+	for _, name := range names {
+		requested[name] = true
+	}
```

Comment:
Maybe add a comment saying that names may contain snap names (because it is inst.Names, i.e. user input), but they will not be matched below.

Reply:
Done